cmd/dexctl/driver.go

```diff
@@ -1,13 +1,12 @@
 package main
 
 import (
-	"github.com/coreos/dex/connector"
 	"github.com/coreos/go-oidc/oidc"
 )
 
 type driver interface {
```

Comment:
I agree with you.I didn't find any reasons other than any historical reasons that I am not aware .That will be cleaner.Let me update the code.